.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Does this mean PRs targeting master branch? I assume we'd want to run on any pull request.

Reply:
Indeed. It's done :)